WBooks/Screens/WBooksNavigationController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  WBooksNavigationController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 23/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class WBooksNavigationController: UINavigationController {
```

Comment:
unnecessary final 🤔

Reply:
I just follow the Gabi's recommendation. Im not sure about the difference

Reply:
When you declare a class as `final`, no other class can inherit from it. It also boosts the overall performance because the compiler does less work looking for the implementation of some methods. I like to add it if I'm sure the class won't have subclasses, but this is a training so it's not really necessary lol

Reply:
Wou, amazing!